apps/console/src/components/pages/protected/organization/billing/pricing-table.tsx

```diff
@@ -9,14 +11,62 @@ declare global {
     }
   }
 
-function PricingPage() {
-  return (
-    <stripe-pricing-table
-      pricing-table-id={stripePricingTableId}
-      publishable-key={stripePublishableKey}
-    >
-    </stripe-pricing-table>
-  );
+interface StripePricingTableProps {
+  pricingTableId: string;
+  publishableKey: string;
 }
 
-export default PricingPage;
\ No newline at end of file
+const StripePricingTable: React.FC<StripePricingTableProps> = ({
+  pricingTableId,
+  publishableKey,
+}) => {
```

Comment:
these aren't used, there is an import at the top:
```
import { stripePricingTableId, stripePublishableKey } from '@repo/dally/auth';
```

that are used instead

```suggestion
const StripePricingTable: React.FC= ({}) => {
```